CoEpi/domain/model/UnixTime.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+struct UnixTime: Codable {
+    let value: Int64
+
+    static func fromUnixTime(unixTime: Int64) -> UnixTime {
```

Comment:
Confusing: method creates UnixTime object from Int64.  Default initializer can be used instead.

Reply:
Right, forgot to update this.